numerics.tex

```diff
@@ -68,19 +79,30 @@ \subsection{The mpmath library}
 0.100000000000000000000192874985
 \end{verbatim}
 
-Like SymPy, mpmath is a pure Python library.
-Internally, mpmath represents a floating-point number
-${(-1)}^s x \cdot 2^y$ by a tuple $(s, x, y, b)$ where
+Like SymPy, mpmath is a pure Python library.  A design decision of SymPy is to
+keep it and its required dependencies pure Python. This is a primary advantage
+ of mpmath over other multiple precision libraries such as GNU MPFR~\cite{Fousse:2007:MMB:1236463.1236468},
+which is much more feature-rich and fast.  Like SymPy, mpmath is also BSD
```

Comment:
@fredrik-johansson can you push a change to this branch for how you would rather word it (and note that this is in response to a comment by the reviewers, see the rebuttal document).